### PR TITLE
Adopt changed rails syntax file names

### PR DESCRIPTION
**Pull request by themilkman**
Hi @facelessuser !
I just stumbled across this. 

Thanks for your work and this plugin!

Cheers
Erik

### CODE REVIEW

File: ApplySyntax.sublime-settings [hunk 1]
```diff
@@ -73,15 +73,15 @@
...
+            "syntax": "Rails/HAML",
             "extensions": ["haml"]
         },
         {
             // This is an example of using a custom function to decide whether or
             // not to apply this syntax. The source file should be in a plugin folder.
             // "name" is the function name and "source" is the file in which the
             // function is contained. The rule should be specified as such.
-            "syntax": "Rails/Ruby on Rails",
+            "syntax": "Rails/Ruby (Rails)",
```
Comment (facelessuser) [inline ApplySyntax.sublime-settings:84]: Same here:

```js
"syntax": ["Rails/Ruby (Rails)", "Rails/Ruby on Rails"],
```

File: ApplySyntax.sublime-settings [hunk 2]
```diff
@@ -73,15 +73,15 @@
             "extensions": ["html.erb"]
         },
         {
-            "syntax": "Rails/Ruby Haml",
+            "syntax": "Rails/HAML",
```
Comment (facelessuser) [inline ApplySyntax.sublime-settings:76]: We may need to keep the old one as a fallback for people using old versions of Sublime. At least until I'm ready to cut them off.

```js
"syntax": ["Rails/HAML", "Rails/Ruby Haml"],
```

**Comment by facelessuser** [comment]
Documentation failures can be ignored. I need to update the spelling exclusion rules as they haven't been updated after a specific documentation theme update.

File: ApplySyntax.sublime-settings (hunk 2 above)
Comment (themilkman) [reply ApplySyntax.sublime-settings]: Ha, I didn't know this is possible, great! Thanks:) Obviously much better.
Just force-pushed with these changes.

Comment (facelessuser) [reply ApplySyntax.sublime-settings]: Perfect! I don't use Ruby, so always appreciative of those who do who are willing to help keep the rules relevant.

**Comment by facelessuser** [comment]
@gir-bot lgtm

**Pull request merged**